src/main/java/io/rcktapp/api/service/ext/RedisHandler.java

```diff
@@ -0,0 +1,171 @@
+package io.rcktapp.api.service.ext;
+
+import java.time.Duration;
+import java.util.Map;
+import java.util.TreeMap;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.forty11.j.J;
+import io.forty11.web.js.JS;
+import io.forty11.web.js.JSObject;
+import io.rcktapp.api.Action;
+import io.rcktapp.api.Api;
+import io.rcktapp.api.Chain;
+import io.rcktapp.api.Endpoint;
+import io.rcktapp.api.Handler;
+import io.rcktapp.api.Request;
+import io.rcktapp.api.Response;
+import io.rcktapp.api.SC;
+import io.rcktapp.api.service.Service;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+import redis.clients.jedis.JedisPoolConfig;
+
+/**
+ * @author kfrankic
+ *
+ */
+public class RedisHandler implements Handler
+{
+   Logger                log               = LoggerFactory.getLogger(RedisHandler.class);
+
+   private String        host              = "";
+   private int           port              = 0;
```

Comment:
```suggestion
   private int           port              = 6379;
```

Default the port to the standard redis port.